study-builder/studies/mpc/snippets/question-therapies-groups.conf

```diff
@@ -0,0 +1,159 @@
+{
+  "question": {
+    include required("../../snippets/picklist-question-multi-list.conf"),
+    "stableId": "THERAPIES",
+    "promptTemplate": {
+      "templateType": "HTML",
+      "templateText": "$therapies_prompt",
+      "variables": [
+        {
+          "name": "therapies_prompt",
+          "translations": [
+            {
+              "language": "en",
+              "text": "For your advanced prostate cancer (prostate cancer that is outside of the prostate), please check off all therapies that you have previously received or are currently receiving (Check all that apply)"
+            }
+          ]
+        }
+      ]
+    },
+    "groups": [
+      {include required("picklist-group-question-therapies-hormones.conf")},
+      {include required("picklist-group-question-therapies-chemo.conf")},
+      {include required("picklist-group-question-therapies-other.conf")},
+
+      {
+        "stableId": "EXPERIMENTAL",
+        "questionType": "PICKLIST",
+        "renderMode": "LIST",
+        "selectMode": "SINGLE",
+        "validations": [],
+        "hideNumber": false,
+        "isDeprecated": false,
+        "isRestricted": false,
+        "picklistLabelTemplate": null,
```

Comment:
Hold on... This object looks like a question object... You don't need these properties... A group object only has these properties:
```
stableId
nameTemplate
options
```